app-android-xamarin/Moodis/Moodis/Feature/Group/GroupActivity.cs

```diff
@@ -0,0 +1,96 @@
+﻿
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Support.V7.App;
+using Android.Widget;
+using Moodis.Constants.Enums;
+
+namespace Moodis.Feature.Group
+{
+    [Activity(Label = "Groups")]
+    public class GroupActivity : AppCompatActivity
+    {
+        readonly GroupActivityModel groupActivityModel = new GroupActivityModel(); 
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+            SetContentView(Resource.Layout.activity_group);
+            InitialiseInputs();
+        }
+
+        public override void OnBackPressed()
+        {
+            base.OnBackPressed();
+        }
```

Comment:
You literally override the method with itself :DD. Just add finish() to close the activity.